gitensite/apps/bookinfo/management/commands/load_repos.py

```diff
@@ -26,10 +26,13 @@ def handle(self, rdf_library, start=0, *args, **kwargs):
             if int(pg_id)<start:
                 continue
             try:
-                metadata=BookMetadata(Book(book_id=pg_id), rdf_library=rdf_library, enrich=should_enrich)    
-                addBookFromYaml(metadata.__unicode__())
+                metadata=BookMetadata(Book(book_id=pg_id), rdf_library=rdf_library, enrich=should_enrich)
+                
+                # Add repo_name to yaml
+                yaml = metadata.__unicode__() + "\nrepo_name: " + repo_name
+                addBookFromYaml(yaml)
```

Comment:
note that your code takes a dict, converts it to a yaml string, and then the first line in addBookFromYaml converts the yaml string into a dict

Reply:
Ok, I see the issue here. We will work on restructuring this.